src/main/java/com/thealgorithms/backtracking/TowerOfHanoi.java

```diff
@@ -0,0 +1,32 @@
+package com.thealgorithms.backtracking;
+/*The problem consists of three rods and a number of disks of different sizes, which can be slid onto any rod. The puzzle starts with the disks in a neat stack in ascending order of size on one rod, the source rod, with the largest disk at the bottom and the smallest at the top. The goal is to move the entire stack to another rod, the target rod, subject to the following rules:
+
+Only one disk can be moved at a time.
+A disk can only be placed on top of a larger disk or an empty rod. */
+
+public class TowerOfHanoi {
```

Comment:
Let's make it a _proper utility class_.

```suggestion
public final class TowerOfHanoi {
    private TowerOfHanoi() {
    }
```